File: problems/2275. Largest Combination With Bitwise AND Greater Than Zero/code/code.cpp
class Solution {
public:
    int largestCombination(vector<int>& candidates) {
        int result = 0;
        for(int i=0; i<31; i++){
            int sum = 0;
            for(int k:candidates){
                if(k & 1<<i) sum++;
            }
            result = max(sum, result);
        }
        return result ;
    }
};

File: problems/2275. Largest Combination With Bitwise AND Greater Than Zero/code/code.py
   class Solution:
    def largestCombination(self, candidates: List[int]) -> int:
        result = 0
        for i in range(31):
            count = 0
            for k in candidates:
                if k & (1 << i):
                    count += 1
            result = max(result, count)
        return result
